target/linux/mediatek/image/mt7622.mk

```diff
@@ -74,7 +74,7 @@ define Device/bananapi_bpi-r64
   DEVICE_VENDOR := Bananapi
   DEVICE_MODEL := BPi-R64
   DEVICE_DTS := mt7622-bananapi-bpi-r64
-  DEVICE_DTS_OVERLAY := mt7622-bananapi-bpi-r64-pcie1 mt7622-bananapi-bpi-r64-sata
+  DEVICE_DTS_OVERLAY := mt7622-bananapi-bpi-r64-pcie1 mt7622-bananapi-bpi-r64-sata mt7622-bananapi-bpi-r64-qorvo-dw3000
```

Comment:
We should find a better way to include multiple DTS overlays, not using bootconf but using U-Boot's Extension board management sub system instead. Because now we'd need 4 configs, pcie1, sata, pcie1+dw3000, sata+dw3000 which will multiply further for every new feature we add...

Reply:
okay

Reply:
I will have a look how I can do that.

Reply:
I found a good example of loading dynamic DT overlay here:
https://www.96boards.org/documentation/consumer/dragonboard/dragonboard410c/guides/dt-overlays.md.html#3-load-overlays-dynamically

As adding the spi device is not needed for early boot it should be possible to just load the DTB in Linux at runtime (SATA or NVME storage in PCIe slot for example could be used for extroot hence DT overlay needs to be applied early by U-Boot).

We should define a nice way to do that, ie. how to package dt-overlay packages in OpenWrt and have the overlay(s) applied at boot (in the right order).

Reply:
@dangowrt Just saw you enabled: https://github.com/openwrt/openwrt/commit/401d7ebf2c9bdcc7c1e24900ed9f8acf87d932c8 :)

Reply:
Yes, and I had your case in mind.
Now you should be able to simply load dt overlay at runtime after mounting configfs and placing it there.

Reply:
I also found your discussion here:
https://patchwork.ozlabs.org/project/openwrt/patch/YZ7hf0DVfM7LSobG@makrotopia.org/

However, it is not working for me. Did I miss anything?

Reply:
Ah okay,
> However, if this whole approach is a dead-end and will just burden us
with additional maintainance in the long run, we have to find a better
way.

Sad :(